### PR TITLE
Percona db server

### PR DESCRIPTION
**Pull request by kayiwa**
this expects three nodes to install a percona xdb cluster.

### CODE REVIEW

File: molecule/default/playbooks.yml [hunk 1]
```diff
@@ -27,3 +27,5 @@
     - role: pulibrary.nginxplus
     - role: pulibrary.rails-app
     - role: pulibrary.pas
+    - role: pulibrary.pas_code
```
[the remark this answers is not in the document]
Comment (kayiwa) [reply molecule/default/playbooks.yml]: that was me trying to determine who is burdened with the merge conflict problem. You or I?

**Comment by carolyncole** [comment]
I'm seeing linting errors
```
  [306] Shells that use pipes should set the pipefail option
    /projects/princeton_ansible/molecule/default/roles/pulibrary.perconaxdb/tasks/secure_install.yml:2
    Task/Handler: Get temporary root password
    
    [305] Use shell only when shell functionality is required
    /projects/princeton_ansible/molecule/default/roles/pulibrary.perconaxdb/tasks/secure_install.yml:11
    Task/Handler: Set the MySQL root password
    
    [306] Shells that use pipes should set the pipefail option
    /projects/princeton_ansible/molecule/default/roles/pulibrary.samba/tasks/main.yml:74
    Task/Handler: Create Samba users if they don't exist yet
```

**Comment by kayiwa** [comment]
roles/pulibrary.perconaxdb/.ansible-lint this is supposed to have muted the lint errors. I'm assuming you ran this from root of the repo?

**Comment by carolyncole** [comment]
@kayiwa I was able to test from the role directory just fine, but at the top level I could not.  I'm guessing this is ok.

**Pull request merged**